aip/general/0193.md

```diff
@@ -9,34 +9,51 @@ placement:
 
 # Errors
 
-Effective error communication is an important part of designing simple and
-intuitive APIs. Services returning standardized error responses enable API
-clients to construct centralized common error handling logic. This common logic
-simplifies API client applications and eliminates the need for cumbersome
-custom error handling code.
+Effective error communication is an important part of designing simple
+and intuitive APIs. Services returning standardized error responses
+enable API clients to construct centralized common error handling logic.
+This common logic simplifies API client applications and eliminates the
+need for cumbersome custom error handling code.
 
 ## Guidance
 
-Services **must** return a [`google.rpc.Status`][Status] message when an API
-error occurs, and **must** use the canonical error codes defined in
-[`google.rpc.Code`][Code]. More information about the particular codes is
-available in the [gRPC status code documentation][].
-
-Error messages **should** help a reasonably technical user _understand_ and
-_resolve_ the issue, and **should not** assume that the user is an expert in
-your particular API. Additionally, error messages **must not** assume that the
-user will know anything about its underlying implementation.
+Services **must** return a
+[`google.rpc.Status`](https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto)
```

Comment:
`md` style: Use named anchors for consistency with other APIs, e.g.,

```
[`google.rpc.Status`][Status]
```

Instead of,

```
[`google.rpc.Status`](https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto)
```

here and throughout.

Reply:
This is fixed.